packages/sdk/src/waku/waku.ts

```diff
@@ -282,6 +303,30 @@ export class WakuNode implements IWaku {
     });
   }
 
+  public send(
+    codec: ICodec<IDecodedMessage>,
```

Comment:
IMO we should move away from those template types and instead, assume that a developer just want to deal with a `payload` (byte array) and review the easiest way to offer out of the box encryption by enhancing said `payload`.